README.md

```diff
@@ -56,6 +56,8 @@ Meteole allows you to retrieve forecasts for a wide range of weather indicators.
 | Update Frequency | Every 3 hours        | Every 6 hours        |
 | Forecast Range   | Up to 51 hours       | Up to 114 hours      |
 
+*note : the date of the run cannot be more than 4 days in the past. In case, take care about the exeample following.*
```

Comment:
example

Reply:
In case, take care about the exeample following. --> maybe a comment like : In the code bellow, you have to change the run.